MANIFEST.in

```diff
@@ -5,4 +5,4 @@ include unittest_requirements.txt
 include pyproject.toml
 include requirements/requirements.common.txt
 include galaxy_ng/app/webserver_snippets/*
-recursive-include galaxy_ng/app/static/*
+graft galaxy_ng/app/static
```

Comment:
TIL: I was not aware about graft. the `webserver_snippets/*` could be a graft also?

Reply:
yes, just replaced it to use graft